docs/source/resources/development_workflow.rst

```diff
@@ -15,35 +15,132 @@ abstractions and have a general idea of what goes into each repository. If this
 is not the case see here (TODO: Add a useful link).
 
 It is strongly recommended you create a folder named something like 
-``nwx_workspace``. Bare minimum you'll want to clone the 
-``NWChemEx-Project/NWChemEx`` repo into your workspace. If you are developing a
-module that will live in say the ``NWChemEx-Project/SCF`` repo you will also 
-want to clone that repo. For the sake of this page we'll assume you are 
-developing a new SCF guess and thus your workspace will look like:
-
-```
-nwx_workspace/
-|
-|---NWChemEx/
-|
-L---SCF/
-```
+``nwx_workspace``. We'll term this directory your NWChemEx workspace. As a bare 
+minimum you'll want to clone the ``NWChemEx-Project/NWChemEx`` repo into your 
+workspace (TODO: Add a link to the git tutorial). More than likely you will need
+to clone at least one additional repository, the repo where your module/code 
+will live. For sake of argument we'll assume that you are writing a module for
+use with the SCF. In this case you will also need to clone the 
+``NWChemEx-Project/SCF``. Your NWChemEx workspace thus looks like:
+
+.. code-block::
+
+   nwx_workspace/
+   |
+   |---NWChemEx/
+   |
+   L---SCF/
 
 TODO: Better file structure ASCII art.
```

Comment:
You might be inspired by `tree --charset=ascii`

Reply:
Thanks!!! Done!!!